kale/embed/__init__.py

```diff
@@ -0,0 +1,42 @@
+# from .attention_cnn import CNNTransformer, ContextCNNGeneric
+# from .gcn import GCNEncoderLayer, RGCNEncoderLayer
+# from .gripnet import GripNetSuperEdges, GripNetSuperVertex, TypicalGripNetEncoder
+# from .image_cnn import (
+#     ResNet18Feature,
+#     ResNet34Feature,
+#     ResNet50Feature,
+#     ResNet101Feature,
+#     ResNet152Feature,
+#     SmallCNNFeature,
+# )
+# from .linformer import LinearMultiheadAttention, LinearTransformerEncoderLayer
+# from ._mpca import MPCA
+# from .mpca import MPCA
+# from ._mpca_torch import MPCA
```

Comment:
Any conflict if importing two?

Reply:
I simply want to import one here, another version will be kept private. I will change the name (e.g. MPCATorch) if we wan to make two versions public.

Reply:
One is simpler. Possible to specify it as an option (e.g. nn_torch = true/false)?